xlive/Read.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
```

Comment:
why is this called "Read"?

Always document header/impl/class files to what they are exactly. In this case, it is for reading arguments, so ReadArgumentUtility seems reasonable.

Reply:
ok will do